client/src/views/customer-info/CustomerInfo.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import axios from 'axios';
+
+function CustomerInfo() {
+  const [customerInfo, setCustomerInfo] = useState(null);
+  const history = useHistory();
+
+  useEffect(() => {
+    // Fetch the data on load
+    axios
+      .get('http://localhost:3001/customer', {
+        withCredentials: true,
+      })
+      .then((response) => {
+        if (response.data === 'unauthenticated') {
+          history.push('/login');
+        } else {
+          setCustomerInfo(response.data.recordset[0]);
+        }
```

Comment:
It's a little bit confusing having a single /customer route that can return either a string OR an object. I would recommend always returning a specific expected type, and checking for "failures" such as the authentication here using a direct object truthy/falsy check (i.e. `if (res.data)`). Alternatively, return the object with an `err` or `authenticated` attribute if you need more control.